src/confd/yang/infix-system.yang

```diff
@@ -115,6 +125,41 @@ module infix-system {
     base shell-type;
   }
 
+  typedef source-state {
+    type enumeration {
+      enum selected {
+        description "Selected NTP server for synchronization";
+      }
+      enum candidate {
+        description "Accepted by not selected source";
+      }
+      enum outlier {
+        description "discarded by the clustering algorithm";
+      }
+      enum unreachable {
+        description "unreachable or unresponsable";
+      }
+      enum falseticker {
+        description "suspected of providing incorrect time";
+      }
+      enum unstable {
+        description "Too variable or unstable source";
+      }
+    }
+  }
+  typedef source-mode {
+    type enumeration  {
+      enum server {
+        description "Normal NTP server";
+      }
+      enum peer {
+        description "Symmetric peer";
+      }
+      enum local-clock {
+        description "Reference clock";
+      }
+    }
+}
```

Comment:
Is there any reference to the terminology used here? If so, please add a link to RFC and section, for example.  I mostly reacted to "Normal" and "falseticker" ...